src/scripts/main.js

```diff
@@ -1,3 +1,21 @@
 'use strict';
 
-// write your code here
+const populationsHtml
+  = document.getElementsByClassName('population');
+const totalPopultaionHtml
+  = document.getElementsByClassName('total-population')[0];
+const avaragePopultaionHtml
+  = document.getElementsByClassName('average-population')[0];
+
+let totalPopulation = 0;
+let avgPopulation = 0;
+let i = 0;
```

Comment:
Why you need this variable? You can remove it and use length prop of populations array(on 19 row)